packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -88,6 +101,23 @@ export default class QuanticSearchBox extends LightningElement {
         },
       },
     });
+
+    if (
+      !this.disableRecentQuerySuggestions &&
+      this.headless.buildRecentQueriesList
+    ) {
+      this.recentQueriesList = this.headless.buildRecentQueriesList(engine, {
+        initialState: {
+          queries: getItemFromLocalStorage(this.localStorageKey) ?? [],
+        },
+        options: {
+          maxLength: 100,
```

Comment:
why 100 here if we display 7 by default? if the user sets the option to 100 it would display 100 recent queries? That seems a lot x)

Reply:
Basically that's the max number of recent queries that could be considered. But the thing is that not all 100 will be displayed:

- The maximum number of suggestions is set with the public property `numberOfSuggestions` which defaults to 7.
- A lot of recent queries will be filtered out based on the query written on the search box, for example if the user writes `a`, only the recent queries starting with this letter will appear.

Reply:
yeah that makes sense, just wanted to know if instead of 7 the user sets 100 they could see 100 displayed?

Reply:
with the current logic yes, if the user has 100 recent queries and set the property to 100 yeah 100 recent queries will be displayed x) we can limit the value of this property if we want, but in the same time we are only realizing the user's wish to see 100 recent queries so are not necessarily doing anything wrong x)